webpack.config.js

```diff
@@ -6,6 +6,7 @@ var index = require('./lib/index-static.jsx');
 
 module.exports = {
   entry: './lib/main.jsx',
+  devtool: 'source-map',
```

Comment:
this doesn't work for Firefox for some reason and I have to set it to `eval` to get it to work.